src/ops/ops.go

```diff
@@ -126,24 +129,47 @@ func (o *ops) SystemctlAction(action string, args ...string) error {
 	return errors.Wrapf(err, "Failed executing systemctl %s %s", action, args)
 }
 
-var ostreeOutputRegex = regexp.MustCompile(`Imported: (\w+)`)
+func refFilePath(refName string) string {
+	return path.Join(ostreeRepo, "refs/heads", refName)
+}
 
 func (o *ops) importOSTreeCommit(liveLogger io.Writer) (string, error) {
```

Comment:
maybe at some point worth thinking about ostree object?

Reply:
:shrug: I'm not sure what it gains us right now.
Also the not-so-long-term plan I think is to rip all this out and move to bootc so I don't know how much we really need to change.

Reply:
I can put this stuff in a separate file if that makes it a bit more organized